### DISCUSSION TITLE
How to search for specific check "Unchanged translation"?

### DISCUSSION THREAD

**Discussion by Phentaq** [WeblateOrg/weblate#7933] [Q&A] [answered]
Hi,

I want to search for all strings with the "Unchanged translation" check (Source and translation are identical) when using the general search function in Weblate. Please let me know how this can be done.

Thank you!

**Comment by nijel** [comment, marked as answer]
Use `check:same` see https://docs.weblate.org/en/latest/user/search.html.

**Reply by Phentaq** [reply]
Thank you, it worked.

I could not find the `same` keyword in the documentation. Maybe an idea to add it.

**Reply by nijel** [reply]
I just did that in ccd07881bc75d7f0cd22ec471d7792c0e0471a28 ;-)
